HLTrigger/Configuration/tables/Special.txt

```diff
@@ -95,8 +95,6 @@ HLT_ZeroBias_Gated_v*		    #       CMSHLT-2860
 HLT_SpecialHLTPhysics_v*	    #       CMSHLT-2860
 HLT_SpecialZeroBias_v*		    #       CMSHLT-2860
 
-HLT_L1SingleEG10er2p5_v*	    # CMSHLT-2860
-HLT_L1SingleEG15er2p5_v*	    # CMSHLT-2860
```

Comment:
Mh, I think these 2 are in the (online) Splashes menu, so they should remain in `Special` ? (assuming online-Splashes is derived from Special)

Reply:
@Martin-Grunewald, you dealt with https://its.cern.ch/jira/browse/CMSHLT-3184, please comment.

Reply:
They are still in the txt file - they had been in there twice, so I removed the duplicates!

Reply:
Okay. Sorry for the noise !

Reply:
Thanks! I guess this settles it @missirol